src/pr-comments.ts

```diff
@@ -50,22 +53,56 @@ interface ReviewThread {
   }
 }
 
-async function getCurrentBranch(): Promise<string> {
+// Parse command line arguments
+function parseArgs() {
```

Comment:
this is not the prettiest -- I wonder if we should add a argparsing library?  there are some lightweight ones:

arg (https://github.com/vercel/arg) will probably do the trick